.travis.yml

```diff
@@ -78,16 +78,11 @@ matrix:
         - os: linux
           env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.14
 
-        # Do a PEP8 test with pycodestyle
+        # Do a PEP8 test with flake8
         - os: linux
-          env: MAIN_CMD='pycodestyle pyvo --count' SETUP_CMD=''
+          env: MAIN_CMD='flake8 pyvo' SETUP_CMD=''
```

Comment:
💯 for changing this and being more strict, it makes much sense for this package.

Reply:
I don't know enough about Travis to see how to make this target more efficient (if you look at the job it installs loads of packages including texlive and takes far longer than it needs to).

Reply:
yes, those can be cleaned up. also there are a few unused files scattered around. I'll try to open a follow-up PR in the evening or during the weekend.

Reply:
given that there are examples and scripts ourside of the `pyvo` directory I would suggest though to run flake8 for everything (and explicitly list all generated or bundled external files to be skipped)

Reply:
Okay I totally agree everything should be flake8 and would be happy to take that in another PR

Reply:
Actually, `flake8` works without the `pyvo` argument so I can remove that and this will still be fine.

Reply:
yes, that's what I meant.